cmd/ferretdb/main.go

```diff
@@ -92,7 +92,8 @@ var cli struct {
 	Test struct {
 		RecordsDir string `default:"" help:"Testing: directory for record files."`
 
-		DisablePushdown bool `default:"false" help:"Experimental: disable pushdown."`
+		DisablePushdown            bool `default:"false" help:"Experimental: disable pushdown."`
+		EnableExperimentalPushdown bool `default:"false" help:"Experimental: enable experimental pushdown."`
```

Comment:
I don't think we need both "test" and "experimental" in the flag name, and say that experimental flag is experimental in the help text.

```suggestion
		EnableNestedPushdown bool `default:"false" help:"Experimental: enable pushdown for nested fields."`
```